examples/python/projdata_visualisation/BackendTools/STIRInterface.py

```diff
@@ -23,6 +23,7 @@ class ProjDataDims(Enum):
     VIEW_NUMBER = auto()
     TANGENTIAL_POS = auto()
     TIMING_POS = auto()
+    VMAX = auto()
```

Comment:
I understand why for the implementation but should `VMAX` be in the `ProjDataDims` Enums

Reply:
I agree. it's terribly ugly. fix will come!